### ISSUE TITLE
notes_labeled.csv

### ISSUE THREAD

**Issue by Abhinav43** [HeoTaksung/Document-to-Sequence-BERT-using-Sequence-Attention#1]
Thank you for this awesome paper and for open-sourcing the code, I am trying to run the code, I do have mimic-iii data but I am getting errors while running the code. 

Could you please provide a dummy CSV to show how the data is the format in 'notes_labeled.csv' which was used to train the model?

Also it would be really helpful if you could provide the preprocessing script for 'notes_labeled.csv'

Thank you!

**Comment by HeoTaksung** [comment]
Thank you for your attention to my paper and code.

We can't provide a dummy CSV to show how the data is the format in 'notes_labeled.csv', because the MIMIC-III dataset requires access permission.

However, the preprocessing script for 'notes_labeled.csv' can be found at the link below, which is the official github of CAML. (Our preprocessing about "notes_labeled.csv" is the same as up to line 33 of Jupyter in the following link.)

Link: https://github.com/jamesmullenbach/caml-mimic/blob/master/notebooks/dataproc_mimic_III.ipynb

Once again, thank you for your interest in our paper.

May you always be happy.

**Comment by Abhinav43** [comment]
Thank you for your reply. I am able to generate the csv.
Again, Thank you for providing the code and paper, 
I have one small doubt, Regarding the code you provide, In the sequence attention, the output is first passing through the tahn() function and then softmax, but I didn't find any tahn() reference in the paper, could you please provide insights on that? it will be really helpful.

https://github.com/HeoTaksung/Document-to-Sequence-BERT-using-Sequence-Attention/blob/6099ff3ae6de5a3bab7065f2d2e28fce69546d28/D2SBERT.py#L26

Thank you :)

**Comment by HeoTaksung** [comment]
I'm really sorry for the late reply.

As you said, tanh() is not written on our paper.

The experimental result indicated in the paper was done using tanh(), and the notation was wrong in our paper.

tanh() is an activation function that adjusts the output value to a value between 1 and -1.

Since this can reduce the scale difference between values, the learning speed is improved and the noise is reduced, so overfitting can be suppressed.

This is my mistake, thank you so much for your in-depth look.
